console-extensions.json

```diff
@@ -222,6 +222,48 @@
       "required": ["HIDE_STATIC_PIPELINE_PLUGIN_PIPELINES_LIST"]
     }
   },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "tekton.dev",
+        "version": "v1beta1",
+        "kind": "PipelineRun"
+      },
+      "component": { "$codeRef": "pipelineRunsList.PipelineRunsListPage" }
+    },
+    "flags": {
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_PIPELINERUNS_LIST"]
+    }
+  },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "tekton.dev",
+        "version": "v1",
+        "kind": "PipelineRun"
+      },
+      "component": { "$codeRef": "pipelineRunsList.PipelineRunsListPage" }
+    },
+    "flags": {
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_PIPELINERUNS_LIST"]
+    }
+  },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "pipelinesascode.tekton.dev",
+        "version": "v1alpha1",
+        "kind": "Repository"
+      },
+      "component": { "$codeRef": "repositoriesList.RepositoriesListPage" }
```

Comment:
https://github.com/openshift-pipelines/console-plugin/assets/102503482/0b519f3c-0971-42f7-b755-adad78bc2eb3

Design is not similar to existing design and on click of create repository existing design takes to Create Repository form.